function-deployer/app.js

```diff
@@ -44,6 +44,11 @@ const functionsToDeploy = [
         runtime:    'dotnet6',
         handler:    'LambdaPerf::LambdaPerf.Function::Handler',
     },
+    {
+        runtime:    'provided',
```

Comment:
could we use `provided.al2` instead?

Reply:
Ah yup! I think that was the recommend one, my bad.